kroxylicious-integration-test-support/src/main/java/io/kroxylicious/test/client/KafkaClient.java

```diff
@@ -191,7 +191,7 @@ public void close() {
         if (channelCompletableFuture != null) {
             channelCompletableFuture.thenApply(Channel::close);
         }
-        bossGroup.shutdownGracefully();
+        bossGroup.shutdownGracefully(0, 15, TimeUnit.SECONDS);
```

Comment:
15s feels generous as a timeout given the goal is to shut things down quickly, maybe set it 2 or 5s?